CONTRIBUTING.md

```diff
@@ -1,15 +1,21 @@
 # Contributing
-
-- [Local Setup](#-Setup-Without-Docker)
-- [Docker Setup](#-Setup-With-Docker-Compose)
-
-## ⚙ Setup Without Docker
-### Client
+- [Issues](#Issues)
+- [Development Guide](#Development)
+    - [Local Setup](#-Setup-Without-Docker)
+    - [Docker Setup](#-Setup-With-Docker-Compose)
+- [Pull Requests](#Pull-Requests)
+
+## Issues
+If you notice any bugs or features that can be added, please open an issue! If you'd like to work on an issue, please assign yourself to it.
+
+## Development 
+### ⚙ Setup Without Docker
+#### Client
```

Comment:
My initial feeling is also the same as @yehee - that setup stuff can stay in README, whereas issues/PR stuff can go into contribution. But mostly because without the setup, our README is blank. But I don't have too strong a feeling on this, as long as it's clear on the README where setup instructions are located.

Reply:
Eventually we'll have a banner in the README and more of a description of what our project is about.